openbsd-compat/defines.h

```diff
@@ -497,10 +497,6 @@ typedef uint16_t	in_port_t;
 #define FPARSELN_UNESCALL       0x0f
 #endif
 
-#ifndef _PATH_VARRUN
```

Comment:
If defined, this doesn't get run. Thus when configure defines it. this doesn't get run. Exactly as it should be.